vpr/src/main.cpp

```diff
@@ -69,6 +69,8 @@ int main(int argc, const char** argv) {
         auto& timing_ctx = g_vpr_ctx.timing();
         print_timing_stats("Flow", timing_ctx.stats);
 
+        error_if_timing_failed();
```

Comment:
Could this be moved into print_timing_stats (or another timing routine) so it isn't visible at the top level?  Ideally we'd keep the top level flow pretty short.